real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/CameraNode.java

```diff
@@ -45,9 +51,18 @@
     public CameraNode(NodeChannel channel, int period) {
         super(new BuggyBaseNode(channel), period, "Camera_Node");
 
+        statePub = new Publisher(channel.getStatePath());
 
         //setup the webcam
-        List<Webcam> webcams = Webcam.getWebcams();
+        List<Webcam> webcams;
+        try {
+            webcams = Webcam.getWebcams();
+        } catch (Exception e) { // TODO remove bad practice here
```

Comment:
None of this in the repo please (yes I know I committed it but for the sake of the review)